photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -402,7 +402,7 @@ object Driver {
         val logsDir = new Path(params.outputDir, LOGS).toString
         implicit val logger = new PhotonLogger(logsDir, sc)
         // TODO: This Photon log level should be made configurable
-        logger.setLogLevel(PhotonLogger.LogLevelInfo)
+        logger.setLogLevel(PhotonLogger.LogLevelDebug)
```

Comment:
Let's keep it at `Info` by default, lest we generate too much log. We need to implement the TODO on the line just above rather than tweak to `Debug`, IMHO.